cookies-modal.html

```diff
@@ -1,10 +1,17 @@
-<div id="cookies-modal-container">
-  <div id="cookies-modal">
-    <div id="cookies-title">
-      🍪 Improve your experience by accepting our <a href="#YOUR-PRIVACY-POLICY-PAGE" target="_blank">privacy policy</a>.
-    </div>
-    <div id="cookies-modal-buttons">
-      <button id="accept-cookies-button" onclick="document.getElementById('cookies-modal').className='cookies-d-none';">Accept</button> <button onclick="document.getElementById('cookies-modal').className='cookies-d-none';">Reject</button>
-    </div>
-  </div>
-</div>
\ No newline at end of file
+<dialog id="cookies-modal">
```

Comment:
Using dialog to make it use the native top layer of the browser and be better interpreted by screen readers.